Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1867,7 +1867,8 @@ def long_running_execution_command(client: Client, params: Dict):
                 mirror_direction=mirror_direction
             )
 
-        except Exception:
+        except Exception as e:
+            demisto.updateModuleHealth(f'Error occurred during long running loop: {e}')
```

Comment:
Use the same message (with the Exception) in both `demisto.updateModuleHealth` and `demisto.error`